internal/pkg/template/workload.go

```diff
@@ -849,21 +850,22 @@ func (t *Template) parseWkld(name, wkldDirName string, data interface{}, options
 func withSvcParsingFuncs() ParseOption {
 	return func(t *template.Template) *template.Template {
 		return t.Funcs(map[string]interface{}{
-			"toSnakeCase":          ToSnakeCaseFunc,
-			"hasSecrets":           hasSecrets,
-			"fmtSlice":             FmtSliceFunc,
-			"quoteSlice":           QuoteSliceFunc,
-			"quote":                strconv.Quote,
-			"randomUUID":           randomUUIDFunc,
-			"jsonMountPoints":      generateMountPointJSON,
-			"jsonSNSTopics":        generateSNSJSON,
-			"jsonQueueURIs":        generateQueueURIJSON,
-			"envControllerParams":  envControllerParameters,
-			"logicalIDSafe":        StripNonAlphaNumFunc,
-			"wordSeries":           english.WordSeries,
-			"pluralWord":           english.PluralWord,
-			"contains":             contains,
-			"requiresVPCConnector": requiresVPCConnector,
+			"toSnakeCase":                  ToSnakeCaseFunc,
+			"hasSecrets":                   hasSecrets,
+			"hasRuntimeEnvironmentSecrets": hasRuntimeEnvironmentSecrets,
```

Comment:
hasRuntimeEnvironmentSecrets parsing function to check if rdws service Manifest has secrets